vscode/package.json

```diff
@@ -757,25 +757,33 @@
 		"vscode:prepublish": "npm run compile",
 		"compile": "tsc -p ./; node ./esbuild.js",
 		"watch": "tsc -watch -p ./ | node ./esbuild.js --watch",
-		"test": "node ./out/test/runTest.js",
+		"test:unit": "npm run compile && node ./out/test/unit/index.js",
+		"test:coverage": "nyc npm run test:unit",
+		"test": "npm run compile && node ./out/test/integration/runTest.js",
 		"nbcode": "node ./out/nbcode.js",
 		"nbjavac": "node ./out/nbcode.js -J-Dnetbeans.close=true --modules --install .*nbjavac.*",
 		"apisupport": "node ./out/nbcode.js -J-Dnetbeans.close=true --modules --install '(org.netbeans.libs.xerces|org.netbeans.modules.editor.structure|org.netbeans.modules.xml|org.netbeans.modules.xml.axi|org.netbeans.modules.xml.retriever|org.netbeans.modules.xml.schema.model|org.netbeans.modules.xml.tax|org.netbeans.modules.xml.text|org.netbeans.modules.ant.browsetask|.*apisupport.*|org.netbeans.modules.debugger.jpda.ant)' && node ./out/nbcode.js -J-Dnetbeans.close=true --modules --enable .*apisupport.ant",
 		"artifactory:check": "node ./esbuild.js --artifactory-check"
-		},
+	},
 	"devDependencies": {
+		"@istanbuljs/nyc-config-typescript": "^1.0.2",
+		"@types/chai": "^4.3.17",
 		"@types/glob": "^7.1.1",
 		"@types/mocha": "^9.0.0",
 		"@types/node": "^13.11.0",
 		"@types/ps-node": "^0.1.0",
+		"@types/sinon": "^17.0.3",
 		"@types/vscode": "^1.76.0",
 		"@types/vscode-webview": "^1.57.1",
 		"@vscode/codicons": "0.0.29",
 		"@vscode/test-electron": "^2.4.0",
+		"chai": "^4.2.0",
 		"esbuild": "^0.16.17",
 		"glob": "^7.1.6",
 		"mocha": "^9.1.2",
+		"nyc": "^17.0.0",
```

Comment:
could you add some minimal description on libraries we are adding in pr description?